js-library/src/request-verifiable-presentation.ts

```diff
@@ -100,11 +124,24 @@ export const requestVerifiablePresentation = ({
   identityProvider: string;
 }) => {
   const handleFlowFactory = (currentFlowId: FlowId) => (evnt: MessageEvent) => {
-    // Check how AuthClient does it: https://github.com/dfinity/agent-js/blob/a51bd5b837fd5f98daca5a45dfc4a060a315e62e/packages/auth-client/src/index.ts#L504
-    if (
-      evnt.data?.method === "vc-flow-ready" &&
-      !currentFlows.has(currentFlowId)
-    ) {
+    // The handler is listening to all window messages.
+    // For example, a browser extension could send messages that we want to ignore.
+    if (evnt.origin !== identityProvider) {
+      console.warn(
```

Comment:
I would lower this to `info` and reword it a bit (i.e "ignoring message from ..."). This is not something to be alarmed by.

Reply:
I used the same as AuthClient: https://github.com/dfinity/agent-js/blob/a51bd5b837fd5f98daca5a45dfc4a060a315e62e/packages/auth-client/src/index.ts#L504

I believe it makes sense that both behave similar.